docs/rules-engine/how-to-use/README.md

```diff
@@ -0,0 +1,220 @@
+
+Look into the [Going Further](#going-further) section to look for more complex and optimized use cases.
+
+### Step 1: Integrate the service
+
+First, the `RulesEngineRunnerModule` should be imported in your application module (see [showcase application configuration](https://github.com/AmadeusITGroup/otter/blob/main/apps/showcase/src/app/app.module.ts#L102)).
+
+The runner is the core of the rules engine. This is where all the Rulesets will be processed to identify the resulting actions. 
+
+[//]: # (This import should be done automatically with the `ng add @o3r/rules-engine` command - This should be fixed by https://github.com/AmadeusITGroup/otter/issues/1789)
+
+### Step 2: Register the action handlers in the runner
+
+The result of the rules engine is a set of actions applied at the runtime of the application.
+
+The engine is agnostic of the implementation of the different actions in your Rulesets. It allows for extensibility and
+a better tree shaking as you will not import useless code to handle actions you do not need.
+
+This means that you will have to explicitly import the action handling modules and register them inside the rules engine.
+
+You will not find the Otter actions in the `@o3r/rules-engine` package but in the ones of the feature they impact:  
+- `UPDATE_ASSET`: requires the import of `AssetRulesEngineActionModule` from [@o3r/dynamic-content](https://www.npmjs.com/package/@o3r/dynamic-content/)
+- `UPDATE_LOCALISATION`: requires the import of `LocalizationRulesEngineActionModule` from [@o3r/localization](https://www.npmjs.com/package/@o3r/localization/)
+- `UPDATE_CONFIG`: requires the import of `ConfigurationRulesEngineActionModule` from [@o3r/configuration](https://www.npmjs.com/package/@o3r/configuration/)
+- `UPDATE_PLACEHOLDER`: requires the import of `PlaceholderRulesEngineActionModule` and `PlaceholderRequestStoreModule` from [@o3r/components](https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/components/)
+
+Once the modules have been imported by the application, they need to be explicitly registered into the Rules Engine in the
+pages where they will be used. For instance, in the showcase, this is done in the [rules engine page component](https://github.com/AmadeusITGroup/otter/blob/main/apps/showcase/src/app/rules-engine/rules-engine.component.ts#L96),  
+
+Note that in the 'New-York availability in summer rule', we only need to register the `UPDATE_CONFIG` action.
+
+> [!WARNING]
+> In case the Rules Engine results in an unregistered action, a warning will be raised. You will be able to access it
+> thanks to the [rules engine debug tools](#how-to-debug-your-application).
+
+### Step 3: Integrate your facts
+
+Now that the rules engine recognizes the `UPDATE_CONFIG` action, it needs to recognize the `outboundDate` [fact](../README.md#fact)
+which will emit values according to the window size of your user's browser.
```

Comment:
this is not from the 'isMobileDeviceFact'?
I think it should be `which should emit values when the outboundDate changes`

Reply:
I must have been tired :D